percona-monitoring-and-management/manifest.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: PMM
+title: "Percona Monitoring and Management"
+version: "2.33.0"
+maintainer: sergey.pronin@percona.com
+description: "Open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB."
+url: https://www.percona.com/software/database-tools/percona-monitoring-and-management
+category: monitoring
+configuration:
+  PMM_ADMIN_PASSWORD:
+    label: "Percona Monitoring and Management admin password"
+    value: "CIVO:ALPHANUMERIC(30)"
+  SERVICE_TYPE:
+    label: "Kubernetes service type"
+    value: LoadBalancer
```

Comment:
Also add 
required_services:
  - loadbalancer 

in manifest.yaml

Reply:
```
required_services:
  - loadbalancer
```

Reply:
Hey @saiyam1814 - changes were made. Could you please check?

Reply:
Thanks, LGTM